SETUP.md

```diff
@@ -1,6 +1,79 @@
 # Running the reference implementation locally
 Running the reference implementation locally can be done in various ways, but the easiest is via Docker Compose.
 
+## Prerequisites
+In order to run the reference implementation the following is needed:
+* JDK 17
+* Maven 3
+* Docker
+* Docker Compose
+* Github PAT to access DCSA artifacts
+
+### Access to Github DCSA artifacts
+
+To use Github Packages, Github Authentication must be set up, even if the packages are public (This seems to be limitation on GitHubs end)
+This is done like this:
+
+To do this, follow these steps:
+
+#### Creating a Github Personal Access Token
+Go to https://github.com/settings/tokens and click "Generate new token"
+Write a note for the token, for example "MAVEN_PACKAGES_PAT"
+Select read:packages, delete:packages, and write:packages.
+Click Generate Token
+Copy the token, we will need it soon. If you leave this page without copying the token, you will need to generate a new one
+Note: Treat this token as you would treat any password.
+
+#### Setting up settings.xml
+
+1. Go to your maven directory. Default directory on Windows is C:\Users\USERNAME\.m2 . If you don't have this directory, you may need to install Maven first.
```

Comment:
I think we should `create it first`. Installing maven will not create the directory (you have to run it first, which in turn will fail if you have not done this series of steps)

Reply:
fixed.